### ISSUE TITLE
Add zrot to LAPACKE

### ISSUE THREAD

**Issue by kevinsung** [Reference-LAPACK/lapack#962]
[zrot](https://github.com/Reference-LAPACK/lapack/blob/master/SRC/zrot.f) is currently missing from LAPACKE. It's not listed in the [header file](https://github.com/Reference-LAPACK/lapack/blob/master/LAPACKE/include/lapack.h).

**Comment by kevinsung** [comment]
I'm interested in tackling this issue. Can a maintainer confirm that it makes sense, and possibly offer some guidance on the steps required? I'm new to this repository.

**Comment by langou** [comment]
Yes, it would make sense to create the LAPACKE wrapper for CROT and ZROT. 
We have added wrappers to LAPACKE organically based on users' demands. 
If you have bandwidth to contribute it, this is great, please submit a PR.
WRT guidance on first steps, I do not have a good pointer handy. 
Any documentation is a tat out-of-date as of now.
Please feel welcome to ask any questions on this thread.
